opencga-app/app/scripts/azure/arm/README.md

```diff
@@ -1,11 +1,64 @@
+# ARM Templates and Deploying OpenCGA to Azure
+
+This document contains information related to the deployment of OpenCGA to Azure using ARM automation scripts.
+
+## Deploy to Azure
+
 <a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fmarrobi%2Fopencga%2Ffeature%2Fazure-hdinsights%2Fopencga-app%2Fapp%2Fscripts%2Fazure%2Farm%2Fazuredeploy.json" target="_blank">
     <img src="http://azuredeploy.net/deploybutton.png"/>
 </a>
 
-## Notes
+## T-Shirt Sizing
+
+The ARM templates defined here support three "t-shirt-sized" deployments. Each of these sizes defines properties such as the number of HDInsight master nodes, the size of VMs, the types of disks those VMs use etc. While it's possible to tweak each of these properties independently, these t-shirt sizes should give you some decent defaults.
+
+The sizes are:
 
-### Avere
+- Small: Useful for small teams, or individuals.
+- Medium: A decent default for most installs that need so support a team of researchers
+- Large: A configurartion that should support a large organisation
+
+Here are the properties that are defined for each t-shirt size:
+
+Component | Property | Small | Medium | Large
+-- | -- | -- | -- | --
+Avere | use-azure | FALSE | TRUE | TRUE
+  | node-quantity | n/a | 3 | 12
+  | ssd-per-node | n/a | 4 | 4
+  | ssd-size | n/a | 256GB | 1TB
+  | node-size | n/a | D16 | E32
+  | premium-blob | n/a | ??? | ???

```

Comment:
I think we should remove this for now as we don't know if its possible so could create confusion

Reply:
Done